src/test/unit/engine/InputTest.cpp

```diff
@@ -119,20 +119,17 @@ TEST(InputTest, SplitterMap) {
   // test prepare
   {
     // set in2 to all zeroes
-    const ArrayBase *ai2 = &(in2->getData());
-    Real64 *idata = (Real64 *)(ai2->getBuffer());
-    for (UInt i = 0; i < 64; i++)
-      idata[i] = 0;
+    ArrayBase *ai2 = &(in2->getData());
+    ai2->zeroBuffer();
 
     // set out1 to all 10's
-    const ArrayBase *ao1 = &(out1->getData());
-    idata = (Real64 *)(ao1->getBuffer());
-    for (UInt i = 0; i < 64; i++)
-      idata[i] = 10;
+    ArrayBase *ao1 = &(out1->getData());
+    Real64 *ptr = (Real64*)ao1->getBuffer();
+    for (size_t i = 0; i < ao1->getCount(); i++)
+      *ptr++ = 10;
```

Comment:
nit, imho the array index[i] was more readable

Reply:
> nit, imho the array index[i] was more readable

Depends if you come from the C world...  I will change it.